packages/medusa-js/src/resources/auth.ts

```diff
@@ -17,6 +18,15 @@ class AuthResource extends BaseResource {
     const path = `/store/auth`
     return this.client.request("POST", path, payload, {}, customHeaders)
   }
+  
+  /**
+   * @description Removes authentication session
+   * @return {ResponsePromise<StoreDeleteAuthRes>}
```

Comment:
Is this empty object needed in the response? What do you think about removing it completely?

Reply:
Would `ResponsePromise<void>` be more appropriate?